juniper_official/README.md

```diff
@@ -1 +1,16 @@
-Rules by juniper employees go here
+# Healthbot default KPIs
+#
+ 
+Contains readily consumable healthbot playbooks and rules which are specific to chassis, linecard, system, interface and protocols
+key performance indicators(KPIs). KPI rules collects the statistics from network devices then analyzes the data and act.
+
+KPI playbook is set of rules, each rule is defined with set of KPIs.
```

Comment:
Rephrase to "The KPI playbook is a set of rules, where each rule is defined with a set of KPIs."

Reply:
corrected and commited